### PR TITLE
ref(stories): add t and tct to pageBanner story

### PR DESCRIPTION
**Pull request by michellewzhang**

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
These stories aren't "public", so we probably don't want these in here?

**Comment by michellewzhang** [comment]
> These stories aren't "public", so we probably don't want these in here?

Ah gotcha

File: static/app/components/replays/pageBanner.stories.tsx [hunk 1]
```diff
@@ -101,21 +108,24 @@ export default storyBook(PageBanner, story => {
...
+            {tct(
+              'Build new products faster by exploring reusable the UI components available inside Sentry. [link]',
+              {
+                link: (
+                  <ExternalLink href="https://sentry.io/orgredirect/organizations/:orgslug/stories">
+                    {t('See stories.')}
+                  </ExternalLink>
+                ),
+              }
+            )}
```
Comment (scttcper) [inline static/app/components/replays/pageBanner.stories.tsx:120]:
```suggestion
            {tct(
              'Build new products faster by exploring reusable the UI components available inside Sentry. [link:See stories]',
              {
                link:  <ExternalLink href="https://sentry.io/orgredirect/organizations/:orgslug/stories" />,
              }
            )}
```

**Review by scttcper — APPROVED** [review]

**Comment by scttcper** [comment]
oh right yeah don't translate these

**Pull request closed without merging**